Quid: Re-add legacy theme

vkareh opened the pull request:
Adding legacy themes back here:
- Quid: Icon theme and window decorations. Used to be called Crux.
- Fog: Icon theme and window decorations. Used to be called Mist.
- Glider: Window decoration theme based on Industrial and SmoothBox.
- Aldabra: Window decoration theme that mimics a really old version of Adwaita.

The `gtk2-engines` package provides gtkrc themes for the original Crux and Mist, but no icon files, no window decorations, nor support for GTK widgets. This patch adds the icons and window decorations, but you'll want a different GTK theme to provide the right widget style and colours.

Fog:
![Screenshot at 2025-04-09 15-43-46](https://github.com/user-attachments/assets/47fd9178-51b0-4664-9d89-3ec7906738f2)

Quid:
![Screenshot at 2025-04-09 16-07-41](https://github.com/user-attachments/assets/11b827d6-d8ca-4a0c-9b76-1d58ed17f911)

Glider:
![glider](https://github.com/user-attachments/assets/a129333e-e3d1-4e7c-85cd-26a14069984d)

Aldabra:
![aldabra](https://github.com/user-attachments/assets/fbd9b9b5-d9fc-4e17-b9d6-8bec7499bfce)

vkareh committed:
Quid: Re-add legacy theme

This theme used to be called Crux, later renamed to Quid, and eventually
removed. This patch brings it back with some updated components. It does
not provide a full desktop theme (i.e. no colours or gtk widget
controls), but just the window decorations and icon theme.

Some distros provide a theme called Crux that only provides a gtkrc
file, and so GTK widgets will just default to Adwaita.

lukefromdc reviewed (approved):
This seems to work fine. The Quid icon and window border themes show up in mate-appearance-properties "customize" tab like all the other partial themes that don't have an index.theme file. Both work.

On a quick test I can't vouch for the completeness of the icon theme with some of the newer things in computing, but as we inherit from the MATE icons we should not get any new missing icons

I was even able to test the window border theme by applying it in WCM to Wayfire's experimental metacity decorator plugin and it works. I would have to shut my session down to test applying it in marco, but the parser in the wayfire plugin actually re-uses the same code to read metacity themes so no issue would be expected

lukefromdc commented:
Let's see if we can get a second review

vkareh committed:
Fog: Re-add legacy theme

This theme used to be called Mist, later renamed to Fog, and eventually
removed. This patch brings it back with some updated components. It does
not provide a full desktop theme (i.e. no colours or gtk widget
controls), but just the window decorations and icon theme.

Some distros provide a theme called Mist that only provides a gtkrc
file, and so GTK widgets will just default to Adwaita.

cwendling commented:
#83 suggests those were removed because they were basically useless copies.  Did the original vanished by now?

vkareh commented:
Right, the originals don't exist any longer as far as I know. There's still a gtk2-engines package (at least in Fedora 41, but I see it in the repo for other distros) that contains the gtkrc file for Crux and Mist, along with the corresponding engine source code, but unless you're running gtk2 applications, all it does is fall back to Adwaita.
In any case, this PR only adds the metacity theme (window borders) and the icons, not the gtk theme nor the actual desktop theme configuration (although I created those locally for my own amusement).

vkareh committed:
marco-themes: Add Aldabara and Glider

These themes used to be part of MATE many years ago and were removed
since they were no longer compatible. This patch brings them back fixed
as window decorations (metacity themes).
